Automattic/wp-calypso
# Newsletter: Remove color option from onboarding

## Proposed Changes

- Remove the color option from the Newsletter Setup screen during onboarding.
- Remove all associated code. I decided to be thorough and remove any associated, now-unused code. This includes the AccentColorControl component and two custom hook files. We could do a simpler PR and just remove the control itself, and leave other code in place in case we need it again in the future. 
- Updated launchpad tests to reflect changes.



![color control](https://github.com/Automattic/wp-calypso/assets/21228350/60cdb28b-2a4c-4089-8e74-d16283dd35b1)

## Testing Instructions



We're testing two things: first, the color control does not show and, second, there are no regressive breakages from removing so much code. 

1) Check out this branch and start a newsletter site at http://calypso.localhost:3000/setup/newsletter/intro
2) Confirm the color control no longer shows on the Setup screen, and that the screen otherwise works as expected.
   - The rest of testing will be simpler if you do not choose the paid newsletter option.
3) Go to Launchpad. Confirm that the launchpad screen, task list, and web preview all load as expected.
   - Note we still had to supply a default color for site creation, which is blue. So you'll see blue colors on the web preview.
4) Click the Personalize your newsletter task. This will take you to the 'Post Setup' screen. Confirm that the color control does not show and that the screen otherwise works as expected. 
5) Go back to launchpad, click the Start Writing task and publish a post. This should launch the site. Confirm that you can access My Home (you are not redirected to Launchpad). 
6) Confirm Launchpad tests still pass by running `yarn run test-client client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/` (I updated tests because we removed some params from a key method).

## Pre-merge Checklist
